reana_server/rest/workflows.py

```diff
@@ -1962,13 +1965,13 @@ def get_workflow_disk_usage(workflow_id_or_name):  # noqa
         summarize = bool(parameters.get('summarize', False))
         block_size = parameters.get('block_size')
         reana_fs = fs.open_fs(SHARED_VOLUME_PATH)
-        if reana_fs.exists(workflow.get_workspace()):
+        if reana_fs.exists(workflow.workspace_path):
             absolute_workspace_path = reana_fs.getospath(
-                workflow.get_workspace())
+                workflow.workspace_path)
```

Comment:
We can now upgrade the minimal version of `reana-db` to `0.7.0.dev20200205` in `setup.py`.